trustmanager/keyfilestore.go

```diff
@@ -9,60 +9,64 @@ import (
 
 const (
 	keyExtension = "key"
+	aliasExtension = "alias"
 )
 
 // KeyStore is a generic interface for private key storage
 type KeyStore interface {
 	LimitedFileStore
 
-	AddKey(name string, privKey data.PrivateKey) error
+	AddKey(name, alias string, privKey data.PrivateKey) error
 	GetKey(name string) (data.PrivateKey, error)
-	AddEncryptedKey(name string, privKey data.PrivateKey, passphrase string) error
-	GetDecryptedKey(name string, passphrase string) (data.PrivateKey, error)
+	GetKeyAlias(name string) (string, error)
```

Comment:
I noticed that everywhere we call GetKeyAlias is right after a call to GetKey. It might simplify things to make GetKey return the alias as another return value (since it looks up the alias anyway). Then we wouldn't need GetKeyAlias.